client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -273,7 +273,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                         {tableTitle}
                     </Typography>
                 </Grid>
-                <Grid item xs={2} >
+                <Grid item xs={2} style={{marginRight : 'auto'}}>
```

Comment:
export to class

Reply:
same

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1097/commits/432b74c0313aa184af5be53035f90e339d3cfc6d)